pkg/pillar/cmd/zedrouter/probe.go

```diff
@@ -380,7 +389,7 @@ func launchHostProbe(ctx *zedrouterContext) {
 							break
 						}
 					}
-					if foundport {
+					if foundport && ipAddrIsValid(info.LocalAddr) {
```

Comment:
I think one round of the IP address checking is done inside SendOnIntf.

Reply:
ok. i'll remove this check here. thanks.